src/algorithms/digi/PhotoMultiplierHitDigi.cc

```diff
@@ -277,3 +330,8 @@ dd4hep::Position eicrecon::PhotoMultiplierHitDigi::get_sensor_local_position(uin
 
   return pos_transformed;
 }
+
+bool  eicrecon::PhotoMultiplierHitDigi::Noise_Digits(float noiseRate, int timeWindow) const
+{
+  return (m_rngUni() < (noiseRate*timeWindow*dd4hep::ns));
+}
```

Comment:
Does this need to be its own function? It's only called in one place.

Reply:
The reason why I did it in this way is that we may need to add some feature to the function, making it longer and more complex. But I see your point.

Reply:
Sounds good!